hi-ml-histopathology/src/histopathology/datamodules/base_module.py

```diff
@@ -14,25 +15,32 @@
 
 from health_ml.utils.bag_utils import BagDataset, multibag_collate
 from health_ml.utils.common_utils import _create_generator
+from health_ml.utils.wsi_utils import list_data_collate
 
-from histopathology.datasets.base_dataset import TilesDataset
+from histopathology.datasets.base_dataset import SlidesDataset, TilesDataset
+from histopathology.datasets.panda_dataset import PandaDataset
```

Comment:
Why this is needed here? this is a base_module, I would not expect any dataset specific to happen here